servers/src/main/java/tachyon/worker/file/FileSystemMasterClient.java

```diff
@@ -101,4 +103,23 @@ public FileInfo call() throws TException {
       }
     });
   }
+
+  /**
+   * Instructs a worker to persist the files.
```

Comment:
Is this true? This looks more like a heartbeat for the file worker to the file system master.